fastlane/metadata/android/en-US/short_description.txt

```diff
@@ -1 +1 @@
-Simple app to help you plan and manage your savings goals easily.
\ No newline at end of file
+Plan and manage your savings and achieve goals easily
```

Comment:
The overall changes look good to me. However, I think the previous short description made more sense, the "and achieve goals" doesn't fit well here imo, maybe you could make it something like just "Plan and manage your saving goals easily", what do you think?

Reply:
> It's up to you, I just wanted to express that the app gives some motivation. I can revert the change

I think we can rephrase it like:
"Plan, manage, and achieve your savings goals easily."